Sources/PixelEditor/Components/Controls/ControlStackView.swift

```diff
@@ -66,25 +66,58 @@ final class ControlStackView : UIView {
   private var subscribers: [UIView & ControlChildViewType] = []
 
   private var latestNotifiedEdit: EditingStack.Edit?
-
+  
   func push(_ view: UIView & ControlChildViewType) {
```

Comment:
Could you add `animated` option to this?
```suggestion
  func push(_ view: UIView & ControlChildViewType, animated: Bool) {
```

Reply:
Sure !